src/site/content/en/blog/reporting-api/index.md

```diff
@@ -0,0 +1,660 @@
+reports are sent to the default endpoint.
+
+By contrast, the **other report types** are generated only if an **explicit policy** is
+set up. They'll be sent to the endpoint that's specified by the policy via the `report-to`
+directive.
+
+{% Banner 'caution', 'body' %} Getting the syntax right for declaration can be tricky,
+because not all policies use the same header structure. Depending on the policy, the right
+syntax may be `report-to=main-endpoint` or `report-to main-endpoint`. Head over to the
+[demo](https://glitch.com/edit/#!/reporting-api-demo?path=server.js%3A1%3A0) for code
+examples. {% endBanner %}
+
+### Example code
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/reporting-api-demo?path=server.js&previewSize=0"
+    title="Reporting API demo on Glitch"
+    allow="geolocation; microphone; camera; midi; vr; encrypted-media"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+## How-to: Receive reports
+
+When deciding where to send your reports, you have two options:
+
+- Option 1: Use an existing report collector service.
+- Option 2: Build and operate your own reporting collector.
+
+**Also note that can set multiple endpoints**. This can be handy to separate reports:
+
+- Maybe your analytics provider takes care of some kinds of reports, and you have an
+  in-house solution for the others.
+- Maybe you want to route these reports to different endpoints, depending on their type:
+  `endpoint.example/coop` `endpoint.example/csp` `endpoint.example/default`
+
+### Option 1: Use an existing report collector service
+
+Some examples of report collector services are [report-uri](https://report-uri.com) and
+[uriports](https://uriports.com). If you know of other solutions, please [open an
+issue](https://github.com/GoogleChrome/web.dev/issues) to let us know, and we'll update
+this post!
+
+**🧐 Things to consider when selecting a report collector as a service:**
+
+- Does this collector support all report types? For example, some reporting endpoint
+  solutions support COOP/COEP reports, but not all do.
+- Does the pricing work for you?
+- Is it OK to share your application's URLs with a third-party report collector? Even if
+  the browser strips sensitive information from these URLs, it's possible that sensitive
+  information [gets leaked this way](https://w3c.github.io/reporting/#capability-urls). If
+  this sounds too risky for your application, operate your own reporting endpoint.
+
+### Option 2: Build your own report collector
+
+Building your own server that receives reports isn't that trivial. To get started, you can
+use our lightweight boilerplate. It's built with express and can receive and display
+reports.
+
+- Head over to the [boilerplate report
+  collector](https://glitch.com/edit/#!/reports-endpoint)
+- Click **Remix to edit**
+- You now have your clone ✨ You can customize it for your own purposes.
```

Comment:
```suggestion
1. Head over to the [boilerplate report
  collector](https://glitch.com/edit/#!/reports-endpoint).
- {% Instruction 'remix' %}
3. You now have your clone! You can customize it for your own purposes.
```

Reply:
Tip: Use [instructions components](https://web.dev/handbook/web-dev-components/#instructions) for Glitch instructions (helps with doc maintenance when Glitch changes UI). Also, use numbered lists for steps.